packages/core/src/docdb/explorer/dbClusterNode.ts

```diff
@@ -22,16 +24,23 @@ export class DBClusterNode extends AWSTreeNodeBase implements AWSResourceNode {
     name: string = this.cluster.DBClusterIdentifier ?? ''
     arn: string = this.cluster.DBClusterArn ?? ''
 
-    constructor(readonly cluster: DBCluster) {
+    constructor(readonly cluster: DBCluster, readonly client: DocumentDBClient) {
         super(cluster.DBClusterIdentifier ?? '[Cluster]', vscode.TreeItemCollapsibleState.Collapsed)
         this.contextValue = 'awsDocDBClusterNode'
+        this.tooltip = `${this.name}${os.EOL}Engine: ${this.cluster.EngineVersion}${os.EOL}Status: ${this.cluster.Status}`
     }
 
     public override async getChildren(): Promise<AWSTreeNodeBase[]> {
         return await makeChildrenNodes({
-            getChildNodes: () => {
-                const nodes = this.cluster.DBClusterMembers?.map(instance => new DBInstanceNode(instance))
-                return Promise.resolve(nodes ?? [])
+            getChildNodes: async () => {
+                const instances: DBInstance[] = (await this.client.listInstances([this.arn])).map(i => {
+                    const member = this.cluster.DBClusterMembers?.find(
+                        m => m.DBInstanceIdentifier === i.DBInstanceIdentifier
+                    )
+                    return { ...i, ...member }
```

Comment:
Is it ok that any duplicate properties on `i` or `member` will favor `member`? This might be what you wanted.

Reply:
Since they're the same instance ID, I would think that it's OK to overlap.  I didn't notice anything else that would collide.